setup.py

```diff
@@ -50,6 +50,7 @@
         "elasticsearch_dsl==7.4.0",
         "sphinx==4.0.2",
         "sphinx_rtd_theme==0.4.3",
+        "myst_parser==0.17.0"
```

Comment:
'sphinx==4.5.0',  # readthedocs
        'sphinx-rtd-theme==1.0.0',  # readthedocs
        'myst-parser==0.17.0',  # readthedocs

Order the pkgs name by alphabetical